packages/fxa-auth-server/package.json

```diff
@@ -140,7 +140,7 @@
     "@types/dedent": "^0",
     "@types/hapi__hapi": "^20.0.10",
     "@types/ioredis": "^4.26.4",
-    "@types/jsonwebtoken": "^8.5.1",
+    "@types/jsonwebtoken": "8.5.1",
```

Comment:
Not anything you did, but I find it odd that the types are now set to 8.5.1 and the actual package `jsonwebtoken` is now referencing 9.0.0... Seems like this could cause some bad types... 

I'd like to think the `@types/*` and the corresponding package should be the same.

Reply:
Yes, I agree – we can try to bump the types to match the version.  FWIW, this currently resolves as if pinned at 8.5.1 anyway on `main`:
```
├─ fxa-auth-server@workspace:packages/fxa-auth-server
│  └─ @types/jsonwebtoken@npm:8.5.1 (via npm:^8.5.1)
```

Reply:
That's because of the semversion directive, `^`. That'll stop it from from getting bumped to 9.X, and it'll use the highest 8.X version available, which is 8.5.1.